src/lobby.rs

```diff
@@ -77,11 +86,20 @@ impl Lobby {
     pub async fn new_game(&self, user_id: UserId, rules: ChargingRules) -> GameId {
         let game_id = GameId::new();
         let mut inner = self.inner.lock().await;
-        let mut game = HashSet::new();
-        game.insert(Participant {
+        let mut participants = HashSet::new();
+        participants.insert(Participant {
             player: Player::Human { user_id },
             rules,
         });
+        let timestamp = SystemTime::now()
+            .duration_since(SystemTime::UNIX_EPOCH)
+            .unwrap()
+            .as_millis() as i64;
```

Comment:
is i64 the right type to store these timestamps? I couldn't find  a way to go from i64 -> SystemTime, so I couldn't put the i64s in the DB and rehydrate correctly.

Reply:
i64 is fine. https://docs.rs/chrono/0.4.10/chrono/ is the library to use for more complicated date time usecases.